corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -155,23 +155,25 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
               {% include "app_manager/partials/nav_menu_media.html" with qualifier='case_search-search_label_media_' ICON_LABEL="Search Icon" AUDIO_LABEL="Search Audio" %}
             <!-- /ko -->
           </div>
-          <div class="form-group" data-bind="hidden: inlineSearchActive">
-            <label for="search_again_label" class="{% css_label_class %} control-label">
-              {% trans "Label for Searching Again" %}
-              <span class="hq-help-template" data-title="{% trans "Label for Searching Again" %}"
-                    data-content="{% trans_html_attr "This text is for the search button displayed at the bottom of search results, so users can search again." %}">
-            </label>
-            <div class="{% css_field_class %}">
-              {% input_trans module.search_config.search_again_label.label langs input_name='search_again_label' input_id='search_again_label' data_bind="value: search_again_label" %}
+            {% if not request|toggle_enabled:'SPLIT_SCREEN_CASE_SEARCH' %}
```

Comment:
Nice attention to detail. If this is hidden, we also shouldn't show this text in the translations download: https://github.com/dimagi/commcare-hq/blob/b616afe421992ccc01293154d7e8a75e34eeb137/corehq/apps/translations/app_translations/download.py#L225-L226

Reply:
That makes sense. Does it matter what order those rows in translation downloads are in? Thinking it might be cleaner to do something like 
```
rows = [rows that are always present]
if not toggles.SPLIT_SCREEN_CASE_SEARCH.enabled(domain):
    rows.append(search_again_label)
```
rather than adding a check for it in the middle, but it could be less nice for a user if we're trying to match order to UI.

Reply:
I don't think the order in the download file matters, I'd go for the cleaner code